### PR TITLE
OneLogin intitial

### PR DESCRIPTION
**Pull request by lschober-bw**
OneLogin directory sync article

### CODE REVIEW

File: _articles/directory-connector/onelogin-directory.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% image directory-connector/onelogin/onelogin-step5.png %}
+
+## Configure Directory Connector
+
+1. Launch the Directory Connector desktop application.
+2. Go to the **Settings** tab.
+3. Select **OneLogin** as the directory type.
+4. Enter the **Client ID** and **Client Secret** copied from the previous steps.
+5. Select the correct Region.
+6. Configure sync options.
```
Comment (kspearrin) [inline _articles/directory-connector/onelogin-directory.md]: Add:

> It is recommended to use the **Overwrite existing organization users based on current sync settings.** option with OneLogin.

Comment (lschober-bw) [reply _articles/directory-connector/onelogin-directory.md]: Should I replace the TIP or add it in #6?

**Comment by kspearrin** [comment]
Just add it in 6

**Comment by lschober-bw** [comment]
Requested change made

**Pull request merged**